### ISSUE TITLE
Linux (Manjaro) - Gimp 2.10.32 - Stable Horde - Exception: Input payload validation failed

### ISSUE THREAD

**Issue by adrien-collombat** [blueturtleai/gimp-stable-diffusion#20]
Hi !
I'm having an issue with the stable-horde plugin.
When I try to use text2image, I get the following error message :

```
Une erreur est survenue durant l’exécution de python-fu-stable-horde
Exception: Input payload validation failed

Traceback (most recent call last):
  File "/usr/lib/gimp/2.0/python/gimpfu.py", line 740, in response
    dialog.res = run_script(params)
  File "/usr/lib/gimp/2.0/python/gimpfu.py", line 361, in run_script
    return apply(function, params)
  File "/usr/lib/gimp/2.0/plug-ins/gimp-stable-diffusion-horde.py", line 212, in generate
    raise Exception(message)
Exception: Input payload validation failed
```

Thanks for your help !
Adrien

**Comment by blueturtleai** [comment]
I just tried it and couldn’t reproduce the problem. Which plugin version do you use? You find the version at top of the py file. It should be 1.3.4.

**Comment by adrien-collombat** [comment]
The py script is indeed in version 1.3.4.
I made a few tests and it seems this issue appears with float values for the "Prompt strength" parameter. With integer values it works just fine.

**Comment by blueturtleai** [comment]
Thanks for the update. I will check it.

**Comment by blueturtleai** [comment]
The problem is fixed in version 1.3.5. Now only integer values can be selected for the prompt strength.
